alfred-telemetry-solr/build.gradle

```diff
@@ -18,6 +18,18 @@ repositories {
 
 
 subprojects {
+    repositories {
+	jcenter()
```

Comment:
can we do this without `jcenter` ?

Reply:
Do you have time to review?

Reply:
I'm trying to have no dependency on `jcenter` :-)

Reply:
Jenkins fails without jcenter:
https://jenkins-2.xenit.eu/job/Xenit%20Github/job/alfred-telemetry/job/ALFREDOPS-646/10/console

Reply:
`org.restlet` is available from `mavenCentral()`

Reply:
Do you know why some excellent developers (Laurent) prefer jcenter?

Reply:
JCenter got popular, because it was hip, trendy and bit more stable than the official maven-central/sonatype repos.

But package hijacking is a big issue, see for example https://twitter.com/JakeWharton/status/1073102730443526144

Reply:
Security is important. I have replaced it.

Reply:
@tgeens org.restlet.jee:org.restlet:2.1.1 is not available via mavenCentral:

https://jenkins-2.xenit.eu/job/Xenit%20Github/job/alfred-telemetry/job/ALFREDOPS-646/16/console

Can I put jcenter back?

Reply:
> ... jcenter() [should] always be last, [and] you need to put mavenCentral() before